packages/gestalt/src/FlexItem.js

```diff
@@ -0,0 +1,35 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTypes from 'prop-types';
+import { buildStyles } from './boxTransforms.js';
+import styles from './Flex.css';
+import {
+  AlignSelfPropType,
+  FlexPropType,
+  type AlignSelf,
+  type Flex,
+} from './boxTypes.js';
+
+export type Props = {|
+  alignSelf?: AlignSelf,
```

Comment:
Seems like a good start, I do imagine this list growing over time.

Reply:
The list of props on FlexItem? What else do you think would be needed?

Reply:
@dangerismycat things like flex basis, margins (auto): https://css-tricks.com/the-peculiar-magic-of-flexbox-and-auto-margins/ . For now, let's leave it as is, this is a good start.

Reply:
Sounds good. I'm not familiar with flex-basis, but it seems useful in some situations. We should add the ability to set auto margins soon, but I think that API will need some more thought, since I don't think we want to open up the full `margin` attribute.